### DISCUSSION TITLE
Dexcom follow show events

### DISCUSSION THREAD

**Discussion by manuferro** [NightscoutFoundation/xDrip#2979] [General]
Using the Dexcom follow,
is it possible to show the several Dexcom events? 
(carbo, meal, etc...)
I've seen they are supported into the API

Thanks

**Comment by Navid200** [comment]
We don't have an active developer supporting Dex share.
We are volunteers.  If anyone wants to enhance/support/maintain the Dex share capabilities of xDrip, please open a PR.

**Reply by corneliusroemer** [reply]
Maybe it would be worth marking the feature as possibly broken in the app to warn people that it probably doesn't work?
